shared/cluster.go

```diff
@@ -273,54 +273,95 @@ func SonobuoyMixedOS(action, version string) error {
 	return err
 }
 
-// ParseNodes returns nodes parsed from kubectl get nodes.
-func ParseNodes(print bool) ([]Node, error) {
-	nodes := make([]Node, 0, 10)
+// GetNodes returns nodes parsed from kubectl get nodes.
+func GetNodes(print bool) ([]Node, error) {
```

Comment:
Is this change solving any issues seen with ParseNodes? I'm trying to see if we can keep it simple and small when possible. We can avoid printing the connection error with print boolean false and run it again with print true.

Reply:
i also do agree....but the only thing with that is like if we do not treat the errors separately ( errors in general, and errors because the cluster is down due to upgrades) when we found some "errror", that we will find in all upgrade scenarios , we are going to do what we were doing, return an error or fmt.Errorf() , but its not actually an error, its just the cluster temporarily unavailable. 
and then on the execution itself, even not printing , it will return the controler of the function as error.

But again, this is not a blocking error, the running of the tests will not abort or even quit the test as failed....its just a matter of nor return error or printing it to the logs, when its not .

With we think that here is more important simplicity than cohesion , then we should just mantain as it was though , like retrying get nodes until eventually it succeeds and in this meantime time in the logs will be there like something as failed ( the way is now the main ) 

https://github.com/rancher/distros-test-framework/blob/67d29dbe12b3fe8703d4528a711012ca3db0211f/shared/aux.go#L36C17-L36C17